packages/apr_iconv.rb

```diff
@@ -6,14 +6,14 @@ class Apr_iconv < Package
   version '1.2.2'
```

Comment:
Bump version?

Reply:
So you renamed the binaries too?

Reply:
Can you please confirm no updates to the manifest files occur after a build?

Reply:
Binaries are renamed and reuploaded, same hashes and manifests.